.github/workflows/release-pontos.yml

```diff
@@ -0,0 +1,34 @@
+name: Release openvas-scanner with pontos
+
+on:
+  pull_request:
+    types: [closed]
+  workflow_dispatch:
+
+jobs:
+  build-and-release:
+    name: Create a new release with pontos
+    # If the event is a workflow_dispatch or the label 'make release' is set and PR is closed because of a merge
+    if: (github.event_name == 'workflow_dispatch') || (contains( github.event.pull_request.labels.*.name, 'make release') && github.event.pull_request.merged == true)
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Setting the Reference
+        run: |
+          if [[ "${{ github.event_name }}" = "workflow_dispatch" ]]; then
+            echo "RELEASE_REF=${{ github.ref_name }}" >> $GITHUB_ENV
+          else
+            echo "RELEASE_REF=${{ github.base_ref }}" >> $GITHUB_ENV
+          fi
+      - name: Release with release action
+        uses: greenbone/actions/release@v2
+        with:
+          python-version: "3.10"
+          conventional-commits: true
+          github-user: ${{ secrets.GREENBONE_BOT }}
+          github-user-mail: ${{ secrets.GREENBONE_BOT_MAIL }}
+          github-user-token: ${{ secrets.GREENBONE_BOT_TOKEN }}
+          gpg-key: ${{ secrets.GPG_KEY }}
+          gpg-fingerprint: ${{ secrets.GPG_FINGERPRINT }}
+          gpg-passphrase: ${{ secrets.GPG_PASSPHRASE }}
+          strategy: calendar
```

Comment:
that seems incorrect? although we were calendar based we still stick with the 22.4 notation until we make the switch to proper semantic and even than we will probably either stick with 22.x.x or make the switch to 23.x than.

Reply:
Yeah. We always use the `stable` branch for releases. Therefore we will change it in the backport and leave it `calendar` in the default branch.

Reply:
ah, ok.